modules/standard/BigInteger.chpl

```diff
@@ -2212,6 +2212,23 @@ module BigInteger {
 
 
   // divexact
+
+/*
+Computes ``n/d`` and stores the result in ``bigint`` instance.
+
+``divexact`` is optimized to handle cases where ``n/d`` results in an integer. 
+When ``n/d`` does not produce an integer, this method may produce incorrect results.
+
+:arg n: numerator
+
+:type n: bigint
+
+:arg d: denominator
+
+:type d: bigint
+*/
+
+
```

Comment:
I'm about to merge this anyway, but typically we don't leave space between the docs-comment and the function documented.